### ISSUE TITLE
when no scene is recognised

### ISSUE THREAD

**Issue by vtriet** [ChristopherRogers1991/mycroft-hue#36]
Hi, I just installed your Hue Skill on Picraft (RPi4). It seems to work ok, but I noticed that if Mycroft doesn't recognise the correct scene in a sentence like "hey mycroft, turn on the **mancave** lights" ... then it seems to default to turning on ALL the scenes, which (in my case) also turns on a lot of Hue Powerplug connected equipment, which is really not wanded on most times.
Is there any way to avoid this ? [I already set the default scene to 3, but that doesn't help.] Thanks ! Richard

**Comment by ChristopherRogers1991** [comment]
By "scene," do you mean "group"?

You mentioned setting it to 3 - was that through the web UI, or the config file on disk? If possible, I'd recommend setting the value on disk (in my experience, the web settings can be a little finicky). Also, note that the value can be a number, or the string name, i.e. you should be able to set it to "mancave" to ensure there's not a mismatch between the ID (number) you're using and the name you're expecting to be used.

**Comment by vtriet** [comment]
Hi, thanks for the quick reply ... my Hue App is in dutch, but is is indeed some lights collected together in a 'group' or 'scene'. Thanks for clarifying that the default scene can be a string as well, and I will set this to "mancave" to test, anf try to figure out how to set this in the raspberry instead of Online at mycroft.ai

Thinking about this behaviour ; could I set the default scene to a non-existing number or string ? so I can avoid any lights being turned on if Mycroft doesn't understand the scene/group intended ?

**Comment by ChristopherRogers1991** [comment]
> could I set the default scene to a non-existing number or string ? so I can avoid any lights being turned on if Mycroft doesn't understand the scene/group intended ?

No, as written, this won't work. If the default scene is not found, Mycroft will speak an error message saying that he couldn't find it, and then it will use group 0, which is all lights.

That could potentially be added as a feature, though.

**Comment by vtriet** [comment]
Thx. I personally think that defaulting to group 0 and turning _everything_ ON feels less ideal in an extensive Hue setup, so for now my alternate solution is to create a specific group 'mycroft' with just 1 lamp and use that as the default group in the Skill. (where is that settings file located ?)
I also think it might be a good idea to give audio feedback if no group was detected so if the voice command is like : "_hey mycroft, turn on the mancave lights_" and Mycroft doesn't recognise any particular group mentioned, then a response somethink like "ok, I did not hear any spcific room mentioned, so I'm turning on the lights in the room mycroft"
Does that make sense ?
